packages/splits-sdk/src/client/base.ts

```diff
@@ -466,8 +472,10 @@ export class BaseClientMixin extends BaseTransactions {
 }
 
 export class BaseGasEstimatesMixin extends BaseTransactions {
-  async multicall(multicallArgs: MulticallConfig): Promise<bigint> {
-    const gasEstimate = await this._multicallTransaction(multicallArgs)
+  async multicall({ calls }: { calls: CallData[] }): Promise<bigint> {
```

Comment:
seems like all these multicall changes that got merged in earlier today are getting reverted.